special-pages/pages/duckplayer/src/js/index.js

```diff
@@ -16,7 +16,7 @@ export class DuckplayerPage {
     /**
      * This will be sent if the application has loaded, but a client-side error
      * has occurred that cannot be recovered from
-     * @returns {Promise<import("../../../../types/duckplayer").InitialSetupResponse>}
+     * @returns {Promise<import("../../types/duckplayer.js").InitialSetupResponse>}
```

Comment:
Why does it alternate between `duckplayer.js` and `duckplayer.ts`? (See `special-pages/pages/duckplayer/integration-tests/duck-player.js`)

Reply:
just my editor auto-updating paths. Typescript allows a few formats here, but it WILL fail if any are incorrect, so no need to worry if they are not identical